constitution.md

```diff
@@ -91,17 +92,36 @@ The **Treasurer** shall:
 5. Event costs shall be subject to audit by the membership.
 6. Submit an annual report detailing the society's budget for the past year.
 7. Review the financial records with members of the executive at least once a semester.
-8. Be responsible for overseeing the societal development team (i.e. Head of Societal Development)
+8. Be responsible for overseeing the communications team (i.e., Head of Communications)
 9. Act as a moderator on the Computer Science Discord.
 
 The **Head of Events** shall:
 1. Report to the Vice President
 2. Be responsible for brainstorming events for the benefit of the social and academic life of students.
-3. Manage the entire events team (i.e. Event Coordinators and Event Organizer) by passing information to the organizer to schedule certain events and coordinating with Event Coordinators to ensure that they are attending events.
+3. Manage the entire events team (i.e., Event Coordinators and Event Organizer) by passing information to the organizer to schedule certain events and coordinating with Event Coordinators to ensure that they are attending events.
 4. Be the final say in which events we will host and create polls if we want the students to decide which events they want.
 5. Be responsible for posting events on the CSS website, Discord, and email and work with the Social Media Coordinator to advertise on social media
 6. Act as a moderator on the Computer Science Discord.
 
+The **Head of Communications** shall:
+1. Report to the Treasurer
+2. Be responsible for leading the Communications team (i.e., Merchandising Coordinator, Promotions Coordinator, Design Coordinator)
+3. Be responsible for planning and executing strategies to grow the Computer Science Society.
+4. Be responsible for increasing our income by means of fundraising and promotion.
+5. Be responsible for growing society presence on campus by means of marketing, social media and promoting events.
+6. Act as a moderator on the Computer Science Discord.
+
+The **Head of Technology** shall:
+1. Report to the President
+2. Be appointed via interview process, conducted by executive Computer Science Society executives.
+3. Be responsible for development, debugging, implementation of changes, and maintenance of the Computer Science Society website.
+4. Be responsible for managing the Computer Science Discord including channels, bots, roles, etc.
```

Comment:
I think adding a `Be responsible for all future CSS technology endeavours` or something similar is a good idea

Reply:
not to mention act as a moderator